### PR TITLE
Add sql_clients_daily

### PR DESCRIPTION
**Pull request by relud**
requires https://github.com/mozilla/bigquery-etl/pull/481

### CODE REVIEW

File: dags/main_summary.py [hunk 1]
```diff
@@ -420,6 +420,34 @@
...
+    start_date=datetime(2019, 11, 5),
+    dag=dag)
+
+sql_clients_daily_export = SubDagOperator(
+    subdag=export_to_parquet(
+        table="moz-fx-data-shared-prod:telemetry_derived.clients_daily_v6",
+        destination_table="sql_clients_daily_v6",
+        arguments=[
+            "--filter=submission_date = DATE '{{ds}}'",
+            "--where=submission_date = DATE '{{ds}}'",
```
Comment (jklukas) [inline dags/main_summary.py]: Are both `filter` and `where` needed here?

File: dags/main_summary.py [hunk 2]
```diff
@@ -420,6 +420,34 @@
...
+            "--static-partitions=submission_date_s3={{ds_nodash}}"
+            "--partition-by=sample_id",
```
[the remark this answers is not in the document]
Comment (relud) [reply dags/main_summary.py]: yes

File: dags/main_summary.py (hunk 1 above)
Comment (relud) [reply dags/main_summary.py]: yes

File: dags/main_summary.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (relud) [reply dags/main_summary.py]: no, this is the parquet output, so it needs to match the existing thing. I can make the table have that clustering though.

**Comment by relud** [comment]
CI error is unrelated

**Pull request merged**